manual/thread_safetiness/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Thread safetiness
+layout: single
+sidebar:
+  nav: "manual"
+toc: true
+toc_sticky: true
+---
+
+ROOT classes support one of the following thread-safety levels:
+
+- Thread unsafe: objects must not be used concurrently by different threads, even if objects are not shared among threads.
```

Comment:
@axel @jblomer is there a nomenclature to distinguish between the "objects must not be used concurrently by different threads" case (i.e. they have unprotected mutable data member for example) and the case where there is an underlying singleton (static data member of function static variable) that makes that means that it not even safe to use distinct object in different threads?

Reply:
Isn't this the difference between `conditionally thread safe` and `thread unsafe`?

Or are you looking for a term for classes that don't use unprotected global statics but do contain thread unsafe `const` methods?

Reply:
@pcanal @Axel-Naumann Speaking of terms, while Wikipedia calls it "conditionally thread safe", I actually did not hear this term before; instead I would have called the concept "thread friendly", which I also find describes the concept better.  What do others think?

Reply:
"Thread friendly" isn't a term of art; "conditionally thread safe" is.